email/1.3/modules/ROOT/pages/email-send.adoc

```diff
@@ -1,136 +1,359 @@
-= Sending Emails with the Email Connector
+= Send Emails with the Email Connector Example - Mule 4
 :keywords: email, connector, configuration, smtp, send, smtps
 :page-aliases: connectors::email/email-send.adoc
 
-Email Connector can send messages over SMTP and SMTPS.
+Anypoint Connector for Email (Email Connector) can send messages over SMTP and SMTPS servers using the *Send* operation. The following examples show how to configure Email Connector to send emails and attachments, both in Anypoint Studio and the XML editor:
 
-== Configuring the Connector
+* <<send-email,Send Emails>> +
+Configure the Email Connector *Send* operation to send emails over a SMTPS server.
```

Comment:
"an SMTP server" because when you say the acronym, the S is pronounced like an .